INTHEWILD.md

```diff
@@ -0,0 +1,153 @@
+- [Konfío](http://konfio.mx) [@uis-rodriguez]
+- [Myra Labs](http://www.myralabs.com/) [@viksit]
+- [Nielsen](http://www.nielsen.com/) [@amitNielsen]
+- [Ona](https://ona.io) [@pld]
+- [Peak AI](https://www.peak.ai/) [@azhar22k]
+- [PeopleDoc](https://www.people-doc.com) [@rodo]
+- [Preset, Inc.](https://preset.io) 
+- [Pronto Tools](http://www.prontotools.io) [@zkan]
+- [PubNub](https://pubnub.com) [@jzucker2]
+- [Reward Gateway](https://www.rewardgateway.com) 
+- [ScopeAI](https://www.getscopeai.com) [@iloveluce]
+- [Showmax](https://tech.showmax.com) [@bobek]
+- [source{d}](https://www.sourced.tech) [@marnovo]
+- [Steamroot](https://streamroot.io/)
+- [Tenable](https://www.tenable.com) [@dflionis]
+- [timbr.ai](https://timbr.ai/) [@semantiDan]
+- [Tobii](http://www.tobii.com/) [@dwa]
+- [Tooploox](https://www.tooploox.com/) [@jakubczaplicki]
+- [Whale](http://whale.im)
+- [Windsor.ai](https://www.windsor.ai/) [@octaviancorlade]
+
+### Entertainment
+- [6play](https://www.6play.fr) [@CoryChaplin]
+- [bilibili](https://www.bilibili.com) [@Moinheart]
+- [Douban](https://www.douban.com/) [@luchuan]
+- [Kuaishou](https://www.kuaishou.com/) [@zhaoyu89730105]
+- [TME QQMUSIC/WESING](https://www.tencentmusic.com/) 
+- [Zaihang](http://www.zaih.com/)
+
+### Education
+- [Brilliant.org](https://brilliant.org/) 
+- [Udemy](https://www.udemy.com/) [@sungjuly]
+- [VIPKID](https://www.vipkid.com.cn/) [@illpanda]
+
+### Energy
+- [Airboxlab](https://foobot.io) [@antoine-galataud]
+- [DouroECI](http://douroeci.com/en/) [@nunohelibeires]
+- [Safaricom](https://www.safaricom.co.ke/) [@mmutiso]
+- [Scoot](https://scoot.co/) [@haaspt]
+
+### Healthcare
+- [Amino](https://amino.com) [@shkr]
+- [Living Goods](https://www.livinggoods.org) [@chelule]
+- [Maieutical Labs](https://maieuticallabs.it) [@xrmx]
+- [QPID Health](http://www.qpidhealth.com/) 
+- [TrustMedis](https://trustmedis.com) [@famasya]
+- [WeSure](https://www.wesure.cn/)
+
+### Others
+ - [Grassroot](https://www.grassrootinstitute.org/) 
+ - [komoot](https://www.komoot.com/) [@christophlingg]
+ - [Let's Roam](https://www.letsroam.com/) 
+ - [Twitter](https://twitter.com/) 
+ - [Yahoo!](https://yahoo.com/) 
+ 
+
+## License
+
+```
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+```
```

Comment:
you probably don't need the license here if it's in the comment above.

Reply:
cool removed.
@suddjian please review when you have a sec. thank you!